config/packages/github_api.yaml

```diff
@@ -3,7 +3,9 @@ services:
         arguments:
             - '@Github\HttpClient\Builder'
         calls:
-            - ['authenticate', ['%env(GITHUB_OAUTH_TOKEN)%', null, !php/const \Github\Client::AUTH_HTTP_TOKEN]]
+            # use: !php/const \Github\Client::AUTH_HTTP_TOKEN instead of: 'http_token'
+            # when the yaml parser can deal with custom tags
+            - ['authenticate', ['%env(GITHUB_OAUTH_TOKEN)%', null, 'http_token']]
```

Comment:
Had to do this change because of the yaml-parser, it is bad that it does not support custom tags